virt/install/uninstalling-virt.adoc

```diff
@@ -5,7 +5,7 @@ include::modules/virt-document-attributes.adoc[]
 toc::[]
 
 You can uninstall {VirtProductName} by using the {product-title}
-xref:../../web_console/web-console.adoc#web-console-overview_web-console[web console].
+xref:../../web_console/web-console.adoc#web-console-overview_web-console[web console] or the  xref:../../cli_reference/openshift_cli/getting-started-cli.adoc#cli-getting-started[CLI].
 
```

Comment:
I might add a little information about why you need to delete the different components separately here.

Reply:
When performing the CNV operator uninstallation via the UI, it automatically removes both the subscription and its associated CSV.
When uninstalling via the command-line, the subscription deletion is not enough and the CSV object must be deleted as well in order to complete the uninstallation.
https://docs.openshift.com/container-platform/4.4/operators/olm-deleting-operators-from-cluster.html#olm-deleting-operator-from-a-cluster-using-cli_olm-deleting-operators-from-a-cluster

Reply:
Added a note explaining that CSV must be explicitly deleted in the CLI because the CSV is not deleted when the subscription is deleted by the user.